paasta_tools/cli/cmds/local_run.py

```diff
@@ -652,10 +653,15 @@ def configure_and_run_docker_container(
         docker_hash = docker_url
         docker_pull_image(docker_url)
 
-    # if only one volume specified, extra_volumes should be converted to a list
+    # Warn if the service is attempting to mount non-whitelisted volumes
     extra_volumes = instance_config.get_extra_volumes()
-    if type(extra_volumes) == dict:
-        extra_volumes = [extra_volumes]
+
+    non_whitelisted_volumes = validate_whitelisted_volumes(instance_config)
+    if len(non_whitelisted_volumes) > 0:
+        sys.stdout.write(PaastaColors.yellow(
+            "Warning: Your service is attempting to mount the following extra volumes "
+            "that are not whitelisted:\n\n\t%s\n\n"
```

Comment:
AFAICT this will do `str(non_whitelisted_volumes)` and look like:

```
Warning: Your service is attempting to mount the following extra volumes that are not whitelisted:
    [{'foo': 'bar'}, {'foo': 'baz'}, {'foo': 'foobar'}]
```
Can we print the volumes line-by-line and in a way that's more user-friendly?